unilog.go

```diff
@@ -334,10 +343,12 @@ func (u *Unilog) Main() {
 
 	fileName := u.target
 
-	Stats, _ = statsd.New(StatsdAddress)
+	Stats, _ = statsd.New(u.StatsdAddress)
 
 	Stats.Tags = append(Stats.Tags, fmt.Sprintf("FileName:%s", fileName))
-	Stats.Tags = append(Stats.Tags, strings.Split(statstags, ",")...)
```

Comment:
I thiiiink this check is a no-op? An empty string will split to an empty slice...